coalib/results/result_actions/GeneratePatchesAction.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+from pygments.lexers import guess_lexer_for_filename
+from pyprint.ConsolePrinter import ConsolePrinter
+from pathlib import Path
+
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.parsing.FilterHelper import FilterHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.coala_modes import mode_normal
+from coalib.parsing.DefaultArgParser import default_arg_parser
+
+
+def show_possibilities(console_printer, i, action):
+    console_printer.print('[{:>4}]  {}. Apply patch (\'{}\')'.format('', i,
+                                                                     action))
+
+
+class GBear():
+    def __init__(self, bear: str):
+        self.bear = bear
+        self.name = bear
+
+
+class GeneratePatchesAction(ResultAction):
+    SUCCESS_MESSAGE = 'Patch generated successfully.'
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self,
+              result,
+              original_file_dict,
+              file_diff_dict):
+        """
+        (G)enerate patches
+        """
+
+        section = Section('cli')
+        section.add_or_create_setting(Setting('default_actions',
+                                              '*: ApplyPatchAction'))
+        section.add_or_create_setting(Setting('no_config', True))
+
+        console_printer = ConsolePrinter()
+        possible_options = []
+        log_printer = LogPrinter()
+        bears_app = []
+        args = default_arg_parser().parse_args()
+        args.files = []
+        to_filename = None
+        to_filename = sorted(result.diffs.items())[0][0]
+        language = guess_lexer_for_filename(to_filename, 'Hello').name
```

Comment:
If it doesn't find any language for `to_filename` it should return something, I changed that to `Error, no file found` but it shouldn't happen because coala already found it when the user choosed this action.